src/scripts/migrate.py

```diff
@@ -0,0 +1,93 @@
+"""Slurp migration pipeline
+
+TODO's:
+  - add CLI: look to makefile for what to include
+"""
+import os
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import oaklib
+import pandas as pd
+
+
+# TODO: implement this func:
+# todo: IDs should be int or str? prolly str
+def determine_next_available_mondo_id(min_id: str, mondo_termlist_df: pd.DataFrame) -> str:
+    """Starting from `min_id`, count up and check until finding the next ID."""
```

Comment:
@matentzn I changed what you wrote earlier:
> satrting from min_id, then counting up and checking if it does not already exist.

to: 
> Starting from `min_id`, count up and check until finding the next ID.

Does it sound like I have the right idea?

Reply:
Yes!